docs/source/models/concept_models/vivarium_mncnh_portfolio/concept_model.rst

```diff
@@ -231,22 +231,17 @@ V&V Checks:
     -    
   * - 3
     - % of each facility type have cesarian section capabilities
-    - Public/governmental health facility (62%), private for-profit health facility (60%), at home (0%), private not-for-profit health facility (86%), other (10%)
+    - At home (0%), in hospital(74%), in clinic/low-level facility (53%)
     - EmONC (Ethiopia; filepath saved `on SharePoint <https://uwnetid.sharepoint.com/:w:/r/sites/ihme_simulation_science_team/_layouts/15/Doc.aspx?sourcedoc=%7B63F98143-C6C3-4CF7-BF62-969344726A87%7D&file=ethiopia_data_received_notes.docx&action=default&mobileredirect=true>`_.) 
     - We want these to be location specific, please code accordingly. These are placeholder values for now (extracted from the EmONC Final Report, link in 'Source' column; the 'other' value is made-up), hopefully we will be able to find similar data available for Pakistan and Nigeria.   
   * - 4a
     - Relative risk of c-section on incidence of hemorrhage
     - 2.05 (1.84-2.29)
     - `Pubu et al 2021 <https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7887994/>`_ 
-    - This value is a stand-in from this population-based study in Tibet (see 'Source' column), in which the authors reported an odds ratio rather than a relative risk. With further research and analysis we will likely update this value. Outstanding items: how does c-section need overlap with hemorrhage/OL, what is the RR, how will we implement this with overlaps in total MD impact of facility type
-  * - 4b
```

Comment:
Would it be better to leave this in and put a relative risk of 0.0 so that it's clear what the arrow from C-section to obstructed labor means? Or is it better to take it out completely?

Reply:
True! I think we could include a relative risk of 0.0 for clarity. We can discuss in RT-checkin as needed!